sdks/python/src/opik/api_objects/trace.py

```diff
@@ -218,6 +218,7 @@ class TraceData:
     tags: Optional[List[str]] = None
     feedback_scores: Optional[List[FeedbackScoreDict]] = None
     project_name: Optional[str] = None
+    is_evaluation: bool = False
```

Comment:
Such "fundamental" object as trace shouldn't know about the existing of evaluation concept, which is a higher level concept. Otherwise we'll have logical relation cycle.
I can think of 2 alternatives:
1. Store evaluation traces ids separately, for example in some set container in evaluation namespace
2. Generalize this field to be something like "created_by" instead of "is_evaluation". In that way the trace we'll have an ability to pass trace origins data (might be some integration, pytest plugin, evaluate function etc). It's way more flexible than is_evaluate, still cheap and doesn't make trace entity logically dependent on higher level features which depend on trace.